apps/meteor/app/livechat/server/lib/Livechat.js

```diff
@@ -154,7 +154,7 @@ export const Livechat = {
 			}
 
 			const onlineAgents = await LivechatDepartmentAgents.getOnlineForDepartment(dept._id);
-			if (onlineAgents && onlineAgents.length) {
+			if (onlineAgents && (await onlineAgents.count())) {
```

Comment:
Depending on the priorities inside the squad, would be good to start moving this big file to TS, at least, doing a feature pair TS version of it and then migrating step by step :)

Reply:
Totally agree and already started working on it here -> https://github.com/RocketChat/Rocket.Chat/pull/28720

PS: This function was also converted to typescript in above PR. If possible, please review it whenever you're free so we can start merging it ;)